src/uwtools/rocoto.py

```diff
@@ -0,0 +1,44 @@
+"""
+Support for writing a Rocoto XML using J2Templating
```

Comment:
The use of `J2Template` is an implementation detail that isn't useful information for the person who'd be reading the docstring, which may be translated into HTML documentation. Maybe
```
Support for creating Rocoto XML workflow documents
```
?